src/amitaRJun20/ReplaceSecondLastStringInarray.java

```diff
@@ -0,0 +1,46 @@
+/*Assignement 30 : 18th July 2020
+ 
+Replace second last globant with Technocredits. 
+input-> Hi Globant Hello Globant Hi Globant GM Globant Hi Globant
+
+output->  Hi Globant Hello Globant Hi Globant GM Technocredits Hi Globant*/
+package amitaRJun20;
+
+public class ReplaceSecondLastStringInarray {
+	
+	String replacedSecondLastString(String input, String replaceWord, String replacedWithWord) {
+		int newLength = input.replaceAll(replaceWord, "").length();
+		int totalCount = (input.length() - newLength) / replaceWord.length();
+		String outputString = "";
+		int count = 0;
+		String[] strArray = input.split(" ");
+		if(totalCount<2) {
+			System.out.println("The String has no second last " +replaceWord+" word");
+			return input;
+		}
+		for (int index = 0; index < strArray.length; index++) {
+			if (strArray[index].equals(replaceWord)) {
+				count++;
+			}
+			if (count == (totalCount - 1)) {
+				strArray[index] =replacedWithWord;
+				count++;
```

Comment:
1. instead of count++ you can break loop here as we have replaced the word we need to in string - It will avoid extra iteration
2. No need to get outputString += strArray[index]+" " --> can return String.join(" ", strArray) as the final string, after for loop is completed/breaked.

Reply:
I have tried using break in the loop but in the output string is not adding further after getting the string " Technocredits" 
and giving output as "The Output String is :  Hi Globant Hello Globant Hi Globant GM " which is incorrect...for that i have used count++  to make that adjustment..

Reply:
Thank you for suggesting the string join method :) 👍